### PR TITLE
zh-translation: translate missing operator doc

### PR DESCRIPTION
**Pull request by morvencao**
[ ] Configuration Infrastructure
[ X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/install/standalone-operator/index.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ aliases:
     $ kubectl apply -f https://preliminary.istio.io/operator.yaml
     {{< /text >}}
 
-    This command runs the operator by creating the following resources in the `istio-operator` namespace:
+    这条命令会 `istio-operator` 命名空间中创建以下资源并开始运行 Istio operator :
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/standalone-operator/index.md]: 不通顺。

File: content/zh/docs/setup/install/standalone-operator/index.md [hunk 2]
```diff
@@ -25,12 +25,12 @@ aliases:
...
     {{< /text >}}
 
-    This command runs the operator by creating the following resources in the `istio-operator` namespace:
+    这条命令会在 `istio-operator` 命名空间中创建以下资源并运行 Istio operator :
 
-    - The operator custom resource definition
-    - The operator controller deployment
-    - A service to access operator metrics
-    - Necessary Istio operator RBAC rules
+    - Istio operator 依赖的 custom resource definition
```
Comment (gorda) [inline content/zh/docs/setup/install/standalone-operator/index.md]: operator 自定义资源

File: content/zh/docs/setup/install/standalone-operator/index.md [hunk 3]
```diff
@@ -25,12 +25,12 @@ aliases:
...
+    - Istio operator 对应的控制器
```
Comment (gorda) [inline content/zh/docs/setup/install/standalone-operator/index.md]: operator 控制器 deployment

File: content/zh/docs/setup/install/standalone-operator/index.md [hunk 4]
```diff
@@ -25,12 +25,12 @@ aliases:
...
+    - 访问 Istio operator 指标信息的 service
```
Comment (gorda) [inline content/zh/docs/setup/install/standalone-operator/index.md]: 这四条，统一去掉或者加上 `Istio` 吧

File: content/zh/docs/setup/install/standalone-operator/index.md [hunk 5]
```diff
@@ -25,12 +25,12 @@ aliases:
...
+    - Istio operator 必要的 RBAC 资源
```
Comment (gorda) [inline content/zh/docs/setup/install/standalone-operator/index.md]: operator 必要的 RBAC 规则

File: content/zh/docs/setup/install/standalone-operator/index.md (hunk 2 above)
Comment (morvencao) [reply content/zh/docs/setup/install/standalone-operator/index.md]: done

File: content/zh/docs/setup/install/standalone-operator/index.md (hunk 3 above)
Comment (morvencao) [reply content/zh/docs/setup/install/standalone-operator/index.md]: done

File: content/zh/docs/setup/install/standalone-operator/index.md (hunk 4 above)
Comment (morvencao) [reply content/zh/docs/setup/install/standalone-operator/index.md]: done

File: content/zh/docs/setup/install/standalone-operator/index.md (hunk 5 above)
Comment (morvencao) [reply content/zh/docs/setup/install/standalone-operator/index.md]: done

**Comment by gorda** [comment]
> /test lint_istio.io

@morvencao lint 失败可能是一个外链的问题，等待修复即可，不用担心

**Comment by morvencao** [comment]
@gorda Thanks.

**Pull request merged**